common/tests/unittests/test_setup_outputdir.py

```diff
@@ -28,6 +29,13 @@ def test(self):
         returned_path = setup_outputdir(path, warn_if_exist=True, create_dir=False, path_suffix=None)
         assert str(Path(returned_path)) == str(path)
 
+        # checks that setup_outputdir handles S3 paths correctly
+        with patch("os.makedirs") as mock_makedirs:  # Mock os.makedirs to ensure no local directory is created
+            path = "s3://test-bucket/test-folder"
+            returned_path = setup_outputdir(path, warn_if_exist=True, create_dir=False, path_suffix=None)
```

Comment:
`create_dir` must be `True` to properly test this.

Reply:
Fixed.